packages/core/src/components/ChoiceList/Choice.test.jsx

```diff
@@ -3,22 +3,32 @@ import FormLabel from '../FormLabel/FormLabel';
 import React from 'react';
 import { shallow } from 'enzyme';
 
-describe('Choice', () => {
-  const label = 'George Washington';
+const label = 'George Washington';
+
+function shallowRender(customProps = {}, children = label) {
+  const props = Object.assign(
+    {
+      name: 'foo',
+      value: 'boo'
+    },
+    customProps
+  );
 
+  return {
+    props: props,
+    wrapper: shallow(<Choice {...props}>{children}</Choice>)
+  };
+}
+
+describe('Choice', () => {
   it('accepts a node as innerHTML', () => {
-    const props = {
-      name: 'presidents',
-      value: label
-    };
-    const wrapper = shallow(
-      <Choice {...props}>
-        <p>
-          <strong>Hello</strong> World
-        </p>
-      </Choice>
+    const data = shallowRender(
```

Comment:
Could say `const { wrapper } = ` to keep the other lines the same.

Reply:
oh nice, good point